docs/user-manual.md

```diff
@@ -1,229 +1,263 @@
-User Manual
-===========
+# User Manual
 
-How to create GitHub App
-------------------------
+## Installation
 
-meows is responsible for the registering/deregistering `Pod`s
-to/from a GitHub Actions runner and the controller requires to have a GitHub App
-secret.
+### Preparation
 
-Please create a GitHub App first on the GitHub page following [the official documentation](https://docs.github.com/en/developers/apps/creating-a-github-app).
-Here are the minimal changes from the default setting on the registration page:
+meows depends on the [cert-manager](https://cert-manager.io/docs/). If you are not installing the cert-manager on your Kubernetes cluster, install it as follows:
 
-- Fill **GitHub Apps Name**
-- Fill **Homepage URL**
-- Uncheck `Active` under **Webhook** section
-- Set **Administration** `Read & Write` permission to the repository scope, if you want to use an repository-level runner.
-- Set **Self-hosted runners** `Read & Write` permission to the organization scope, if you want to use an organization-level runner.
+```bash
+$ curl -fsLO https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
+$ kubectl apply -f cert-manager.yaml
+```
 
-Then, you are redirected to the **General** page and what you should do is:
+You need to manually create some secrets and a configmap in the `meows` namespace at the initial deployment.
+So make the `meows` namespace to prepare.
 
-1. Click `Generate a private key` and downloads the generated private key.
-2. Keep `App ID` shown on the top of the page somewhere.
+```bash
+$ kubectl create namespace meows
+```
 
-Next, you should proceed to the Install App page from the sidebar and click the
-`Install` button. You are asked to give the permission to `All repositories`
-or `Only select repositories`. `Only select repositories` is recommended because
-the permission is very strong. You should decide the scope wide enough depending on
-how you use this controller.
+### Creating GitHub Credential Secret
 
-Finally, you should get the installation ID from the URL of the page to which you are
-redirected. The URL should look like `https://github.com/organizations/cybozu-go/settings/installations/12345`
-and `12345` is your installation ID.
+meows supports two ways of GitHub authentication:
 
-How to deploy meows
----------------------------------------
+1. GitHub App
+2. Personal Access Token (PAT)
```

Comment:
Is there any difference between these two ways of authentication?
If not, could you write `there is no difference between these two ways of authentication`.